provider/pdns/pdns_test.go

```diff
@@ -955,27 +956,13 @@ func (suite *NewPDNSProviderTestSuite) TestPDNSConvertEndpointsToZones() {
 	suite.NoError(err)
 	suite.Equal([]pgo.Zone{ZoneEmptyToLongPatch}, zlist)
 
-	// Check endpoints of type CNAME always have their target records end with a dot.
+	// Check endpoints of type CNAME, ALIAS, MX, SRV, and NS always have their values end with a trailing dot.
 	zlist, err = p.ConvertEndpointsToZones(endpointsMixedRecords, PdnsReplace)
 	suite.NoError(err)
 
 	for _, z := range zlist {
 		for _, rs := range z.Rrsets {
-			if rs.Type_ == "CNAME" {
-				for _, r := range rs.Records {
-					suite.Equal(uint8(0x2e), r.Content[len(r.Content)-1])
-				}
-			}
-		}
-	}
-
-	// Check endpoints of type MX and SRV always have their values end with a trailing dot.
-	zlist, err = p.ConvertEndpointsToZones(endpointsMixedRecords, PdnsReplace)
-	suite.NoError(err)
-
-	for _, z := range zlist {
-		for _, rs := range z.Rrsets {
-			if rs.Type_ == "MX" || rs.Type_ == "SRV" {
+			if rs.Type_ == "CNAME" || rs.Type_ == "ALIAS" || rs.Type_ == "MX" || rs.Type_ == "SRV" || rs.Type_ == "NS"
```

Comment:
Thanks for this PR. 
That's a good idea to align with all the supported type 👍.
🤔 Wdyt of going further, put this into a const array and check if rs.Type (& ep.RecordType) is in this const array ?